### PR TITLE
Store and include claimant tz offset in doc upload stamps

### PR DESCRIPTION
**Pull request by markolson**

### CODE REVIEW

File: app/uploaders/claim_documentation/pension_burial/workflow.rb [hunk 1]
```diff
@@ -6,11 +6,13 @@
...
 # Convert uploaded files for Pension/Burial to a format, stamping them twice
-# before transfering them over to the SFTP server.
+# before transferring them over to the SFTP server.
 
 class ClaimDocumentation::PensionBurial::Workflow < Workflow::File
   run Workflow::Task::Shared::ConvertToPdf
-  run Workflow::Task::Shared::DatestampPdfTask, text: 'Vets.gov', x: 0, y: 0
+  # The timestamp in this task is UTC, but the business requirement is for CST.
+  # Datetime will be added to the append text.
+  run Workflow::Task::Shared::DatestampPdfTask, text: 'Vets.gov', x: 5, y: 5, text_only: true
```
Comment (lihanli) [inline app/uploaders/claim_documentation/pension_burial/workflow.rb]: with `text_only: true` wouldn't that omit that timestamp?

Comment (markolson) [inline app/uploaders/claim_documentation/pension_burial/workflow.rb]: The timestamps are added via the `append_to_stamp` option via the `stamp_text` method in the model and subclasses.

Comment (lihanli) [inline app/uploaders/claim_documentation/pension_burial/workflow.rb]: the append_to_stamp text isn't added when text_only is true

Comment (markolson) [inline app/uploaders/claim_documentation/pension_burial/workflow.rb]: Ugh, no, you're right. The logic only does the append if there's a timestamp. Going to change that now.

**Pull request merged**